src/pykeen/datasets/generate_literals.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+"""Generate a fake literals dataset to accompany a real dataset.."""
```

Comment:
Do we still need this script now that we have actual literals?

Reply:
No. Maybe there's a better place to put it not inside the package?